docs/sql-ref-syntax-aux-resource-mgmt-add-jar.md

```diff
@@ -33,15 +33,30 @@ ADD JAR file_name
 
 * **file_name**
 
-    The name of the JAR file to be added. It could be either on a local file system or a distributed file system.
+    The name of the JAR file to be added. It could be either on a local file system or a distributed file system or an ivy URL.
+    Apache Ivy is a popular dependency manager focusing on flexibility and simplicity. Now we support two parameter in URL query string:
 
+     * transitive: whether to download dependent jars related to your ivy URL. It is case-sensitive and only take last one if multiple transitive parameters are specified.
+     * exclude: exclusion list when download ivy URL jar and dependent jars.
+
+    User can write ivy URL such as:
+
+      ivy://group:module:version
+      ivy://group:module:version?transitive=[true|false]
+      ivy://group:module:version?exclude=group:module,group:module
+      ivy://group:module:version?exclude=group:module,group:module&transitive=[true|false]
+        
 ### Examples
 
 ```sql
 ADD JAR /tmp/test.jar;
 ADD JAR "/path/to/some.jar";
 ADD JAR '/some/other.jar';
 ADD JAR "/path with space/abc.jar";
+ADD JAR "ivy://org.apache.hive:hive-contrib:2.3.7";
+ADD JAR "ivy://org.apache.hive:hive-contrib:2.3.7?transitive=false"
```

Comment:
Just a question. Without `transitive=true`, does `hive-contrib` work correctly?

Reply:
> Just a question. Without `transitive=true`, does `hive-contrib` work correctly?

With-hive, can work. change to
```
ADD JAR "ivy://group:module:version";
ADD JAR "ivy://group:module:version?transitive=false"
``` 
?